### PR TITLE
Jobs board UI

### PR DESCRIPTION
**Pull request by marcoacierno**

### CODE REVIEW

File: frontend/src/pages/jobs/index.tsx [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+  const client = getApolloClient();
+
+  await Promise.all([
+    prefetchSharedQueries(client, locale),
+    queryAllJobListings(client, {
+      language: locale,
+    }),
+  ]);
+
+  return addApolloState(client, {
```
Comment (estyxx) [inline frontend/src/pages/jobs/index.tsx:75]: just curious: what it does this `addApolloState`?

File: frontend/src/pages/jobs/index.tsx [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+const JobsBoardPage = () => {
+  const language = useCurrentLanguage();
+  const {
+    data: { jobListings },
+  } = useAllJobListingsQuery({
+    variables: {
+      language,
+    },
+  });
+  console.log("jobListings", jobListings);
```
Comment (estyxx) [inline frontend/src/pages/jobs/index.tsx:28]:
```suggestion
```

**Review by estyxx — APPROVED** [review]
looks good!

File: frontend/src/pages/jobs/index.tsx (hunk 1 above)
Comment (marcoacierno) [reply frontend/src/pages/jobs/index.tsx:75]: It handles sharing the cache between the Apollo client running in the server and on the client: https://github.com/pythonitalia/pycon/blob/f1f399191f98ce8353a0b23ade5b256ca428acbf/frontend/src/apollo/client.ts#L116

This allows us to fetch the data on the server, generate the pages and have everything ready without having the client connect to GraphQl

**Pull request merged**